specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2151,7 +2151,7 @@
           "Entities"
         ],
         "description": "Timeline for an entity.",
-        "operationId": "EntitiesGetTimeline_list",
+        "operationId": "Entities_GetTimeline",
```

Comment:
This is a new API. added last week, so no one used it for now.

Reply:
> > @sagamzu, could you follow the breaking change policy for this one? Please follow the step in http://aka.ms/bcforapi
> 
> @lirenhe this is a new api. added last week and no one use it for now.
> Is it still a breaking change?

Please contact @azbreakchangereview team as they are responsible to decide whether this is a breaking change or not.